src/code/code_800F7260.c

```diff
@@ -60,36 +62,36 @@ u8 D_801333CC = 0;
 
 u16 D_801333D0 = 0;
 
-Vec3f D_801333D4 = { 0.0f, 0.0f, 0.0f };
+Vec3f D_801333D4 = { 0.0f, 0.0f, 0.0f }; // default pos
 
-f32 D_801333E0 = 1.0f;
+f32 D_801333E0 = 1.0f; // default freqScale
 
-UNK_TYPE D_801333E4 = 0;
+UNK_TYPE D_801333E4 = 0; // unused
 
-s8 D_801333E8 = 0;
+s8 D_801333E8 = 0; // default reverbAdd
 
-UNK_TYPE D_801333EC = 0;
+UNK_TYPE D_801333EC = 0; // unused
```

Comment:
unused `UNK_TYPE` stuff can just be `s32`, `UNK_TYPE` is more meant as a placeholder when waiting further documentation